js/ui/map.js

```diff
@@ -166,8 +157,8 @@ var Map = module.exports = function(options) {
     this._setupContainer();
     this._setupPainter();
 
-    this.on('move', this._update.bind(this, false));
-    this.on('zoom', this._update.bind(this, true));
+    this.on('move', this._update);
+    this.on('zoom', this._update);
```

Comment:
Did you intentionally remove `bind`s here? They set the `updateStyle` argument of `_update` (e.g. so that `move` doesn't trigger zoom recalc as far as I remember and `zoom` does). Without `bind`, `updateStyle` will always be truthy because `move` and `zoom` pass event data to listeners.